### PR TITLE
Grids

### PR DESCRIPTION
**Pull request by rmauritsson**

### CODE REVIEW

**Comment by Torres-ssf** [comment]
### Great work team👏
You did great on this project. But before submitting to master, we need to make some changes to make it in line with Microverse guidelines. The changes are:

- [x] It seems that the readme file is missing from your project. The readme file must have a description from your project. Something like this [one](https://github.com/Torres-ssf/nyt-clone/blob/master/README.md)

- [ ]  There are some errors in your CSS code. Check them using this CSS [validator](https://jigsaw.w3.org/css-validator/) . Ignore the warnings.

- Navigation bar

- [ ] Use the `nav` tag for the navbar.

- [ ] The NY Times logo is a link (both of them)

- [x] You set the display from your navbar to be flex. By doing that there is no need to set its children to display float.

- Main Content

- [x]  Its best practice to use the main tag for the most content of your page. In this project case, the most important content is the main article.

- [ ] All images and videos should be enclosed by a `figure` tag with `figcaption` that gives a description of the embedded image or video.

- [ ] Any sections that include text (paragraphs) should be wrapped with a `section` tag. Its best practice to use section tag instead of `div` tags. Div tags have no semantic meaning. Semantic it's a very important topic in HTML. You can learn more about semantic HTML [here](https://developer.mozilla.org/en-US/docs/Glossary/Semantics).

- [ ] You missed the aside section who lives on the right of the main content with the Editor's picks and a Sign-Up button. This section must be inside an `aside` tag.

- Additional Links (section below the main content)

- [x] All images and videos should be enclosed by a `figure` tag with `figcaption` that gives a description of the embedded image or video.

- Footer

- [x] The NYTimes logo is a link

- [ ] It includes a *go to homepage* link on the top right corner of the footer (just like the original page)

Kindly make these changes and submit for code review again. Happy coding.

**Review by Oghenebrume50 — CHANGES_REQUESTED** [review]
Hi team good job :+1:  we still have more work to do though :smiley:  

-Validation the HTML and resolve all warnings and errors
- Setup stickler for this project
- use an aside tag for the Editor's pick section

Please fix these things and request for a re-review. :clinking_glasses:

**Comment by rmauritsson** [comment]
Hey
Seems like everything you are asking us is already done
-we have already validated the file and there are no errors
-we also have already included the editors pick using the aside tag.. Please check well

File: style.css
Comment (bolah2009) [inline style.css:158]: #### `HINT`

Add `z-index` property with an `<integer>` value to the `nav` element. This makes it render on top of elements with `position: absolute`. See current display below:
nav index render

Learn more [here](https://developer.mozilla.org/en-US/docs/Web/CSS/z-index).

File: README.md
Comment (bolah2009) [inline README.md]: #### `HINT`

Add spaces to make a markdown list. Kindly, do this for other list items.

```suggestion
- Embedding images
```

File: README.md
Comment (bolah2009) [inline README.md]: #### `HINT`

Add space to achieve a markdown heading style.

```suggestion
# Collaborators
```

**Review by bolah2009 — CHANGES_REQUESTED** [review]
## Awesome Job @rmauritsson and @KingsleyMcSimon  :clap: Almost there :smile: 


### The [clone](https://raw.githack.com/KingsleyMcSimon/nytproject/grids/index.html) looks nice and uses CSS positioning effectively. Good Job. :ok_hand:  

Before giving a :+1: let's make some **required** changes.
- [REQUIRED] Add appropriate linter file. [See comment below](https://github.com/KingsleyMcSimon/nytproject/pull/1#pullrequestreview-278893356)
   > [Set up a code linter](https://github.com/microverseinc/linters-config/tree/master/css).
- [REQUIRED] [Fix HTML validation warnings/errors](https://validator.w3.org/nu/?doc=https%3A%2F%2Fraw.githack.com%2FKingsleyMcSimon%2Fnytproject%2Fgrids%2Findex.html)
   > Validate your HTML markup with https://validator.w3.org/#validate_by_upload.
- [REQUIRED] Make sure nothing is messed up. [See comment](https://github.com/KingsleyMcSimon/nytproject/pull/1#discussion_r317045089).
- [Next Project] Use GitHub Flow: For HTML & CSS projects, use only one feature branch (e.g. `embedding-images-and-video`, `html-forms`, `responsive-design`, etc) and **do not merge any feature branch to master until it's been approved**.
   > Follow the [Github Flow guidelines](https://microverse.pathwright.com/library/microverse-onboarding/86777/path/step/53987922/).
- [OPTIONAL] Format readme appropriately. See comments for hints. You can add screenshot, project objective, assignment link (odin) and source page.
   > Remember about descriptive README file.

Good luck with your next submission. :star2: 

Cheers
[Bola](https://bolabuari.com/)

###### PS: Note that comments tagged as `OPTIONAL` are not part of the minimum requirements to approve this PR

**Review by bolah2009 — COMMENTED** [review]
#### `REQUIRED`

Add `.stickler.yml` file to the root of the project. Also let's use [this file](https://github.com/microverseinc/linters-config/blob/master/css/.stickler.yml). :point_down: 

``` yml
linters:
  stylelint:
    config: 'stylelint.config.js'
```
- Also, add the `stylelint.config.js` that was referenced above, this can be found [here](https://github.com/microverseinc/linters-config/blob/master/css/stylelint.config.js)

If you are having an issue with setting up linter, read the `Troubleshooting` section of [this repo](https://github.com/microverseinc/linters-config/tree/master/css).

**Comment by bolah2009** [comment]
### Hi @rmauritsson, I hope my review comments answered [your comment here](https://github.com/KingsleyMcSimon/nytproject/pull/1#issuecomment-524094924).

@Oghenebrume50 asked to add linter and fix HTML validator warnings/errors. This has not been done. Check my review comments for more details.
  > Seems like everything you are asking us is already done

Kindly, check [this link](https://validator.w3.org/nu/?doc=https%3A%2F%2Fraw.githack.com%2FKingsleyMcSimon%2Fnytproject%2Fgrids%2Findex.html). There are about 14 warnings on using section tag with an heading element and using `h1` as only a top-level element.
  > -we have already validated the file and there are no errors

I confirmed that `editors pick` is, in fact, an `aside` element. @Oghenebrume50 most have mistaken it with a `div` element as the element is structured this way `body > main > aside.editors-pick > div.wrap` having `div` as its main container. :+1: 
  > -we also have already included the editors pick using the aside tag.. Please check well

File: index.html
Comment (bolah2009) [inline index.html]: #### `REQUIRED`

Consider using `h2-h6` element here instead. `<h1>` should be used only as a top-level heading. 

> Using only h1 elements in a HTML document results in a flat document outline as heading element semantics are conveyed to users as per the numeric in the heading element tag name.

Learn more [here](https://www.w3.org/wiki/HTML/Usage/Headings/h1only)

Comment (bolah2009) [inline index.html]: #### `REQUIRED`

Consider using `h2-h6` element here instead. `<h1>` should be used only as a top-level heading. 

> Using only h1 elements in a HTML document results in a flat document outline as heading element semantics are conveyed to users as per the numeric in the heading element tag name.

Learn more [here](https://www.w3.org/wiki/HTML/Usage/Headings/h1only)

**Pull request closed without merging**
